google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseAdminClient.java

```diff
@@ -178,6 +174,73 @@
    */
   OperationFuture<Backup, CreateBackupMetadata> createBackup(Backup backup) throws SpannerException;
 
+  /**
+   * Creates a copy of backup from an existing backup in a Cloud Spanner instance.
+   *
+   * <p>Example to copy a backup.
+   *
+   * <pre>{@code
+   * String instanceId                  ="my_instance_id";
+   * String sourceBackupId              ="source_backup_id";
+   * String destinationBackupId         ="destination_backup_id";
+   * Timestamp expireTime               =Timestamp.ofTimeMicroseconds(micros);
+   * OperationFuture<Backup, CopyBackupMetadata> op = dbAdminClient
+   *     .copyBackup(
+   *         instanceId,
+   *         sourceBackupId,
+   *         destinationBackupId,
+   *         expireTime);
+   * Backup backup = op.get();
+   * }</pre>
+   *
+   * @param instanceId the id of the instance where the source backup is located and where the new
```

Comment:
This means that the method won't support copying a backup from one instance to another. Is that intentional? I thought that an important part of this feature was to allow copying a backup to a different instance.

Reply:
Ah, I see that the next method does allow copying from one instance to another. Maybe worth a small note in the comments that if a user wants to do that, then they need to use the method that takes a `BackupId` and `Backup` as arguments.

Reply:
Okay sure I will add that. Thanks